peer/bitfield.go

```diff
@@ -20,3 +20,12 @@ func (bf bitfield) set(index int) {
 	}
 	bf[bucket] |= 1 << (7 - index%8)
 }
+
+// unset sets bitfield value at a certain index to false
+func (bf bitfield) unset(index int) {
```

Comment:
I added this function at first before realising that we should not need it at all. Might be fine keeping here just in case.

Reply:
If you keep it, add a test?

Reply:
it has been added precisely to make a test.